server/src/main/java/org/elasticsearch/index/search/QueryParserHelper.java

```diff
@@ -56,6 +57,7 @@
         }
         ALLOWED_QUERY_MAPPER_TYPES.add("scaled_float");
         ALLOWED_QUERY_MAPPER_TYPES.add(TextFieldMapper.CONTENT_TYPE);
+        ALLOWED_QUERY_MAPPER_TYPES.add(AnnotatedTextFieldMapper.CONTENT_TYPE);
```

Comment:
@jpountz this is currently a bit of a blocker for moving annotated_text to a plugin.

Reply:
I agree this one is annoying. I started a discussion at #31655.